src/legacy/server/i18n/index.js

```diff
@@ -26,11 +26,16 @@ import { fromRoot } from '../../utils';
 export async function i18nMixin(kbnServer, server, config) {
   const locale = config.get('i18n.locale');
 
-  // eslint-disable-next-line max-len
-  const translationsDirs = [fromRoot('src/legacy/ui/translations'), fromRoot('src/legacy/server/translations'), fromRoot('src/core/translations')];
+  const translationsDirs = [
```

Comment:
I just realized that this code was written before we decided to keep translations only within `xpack/plugins/translations`. So technically what we should do is to just remove this `translations` const entirely and keep only `config.get('plugins.scanDirs')`. Thoughts?